package.json

```diff
@@ -368,6 +368,12 @@
                 "command": "editor.action.toggleColumnSelection",
                 "intellij": "Column Selection Mode"
             },
+            {
+                "key": "shift shift",
+                "mac": "shift shift",
+                "command": "workbench.action.showCommands",
```

Comment:
Hello @xc1427 @jsanchezgarcia @joaomoreno @brianegan @nosovsh 

I'm going to bind `shift shift` as `Command Palette` (`workbench.action.showCommands`). What do you think?